api/tests/acceptance/application/sco-organization-learners/sco-organization-learner-controller_test.js

```diff
@@ -287,73 +287,6 @@ describe('Acceptance | Controller | sco-organization-learners', function () {
         // then
         expect(response.statusCode).to.equal(204);
       });
-
-      context('when no organizationLearner not linked yet found', function () {
```

Comment:
revoir test unitaire/integration associé à celui-ci

Reply:
`api/tests/integration/domain/usecases/create-and-reconcile-user-to-organization-learner_test.js`
le nom du test d'acceptance porte à confusion mais on a bien un test d'integration qui couvre son cas : 'should nor create nor associate organizationLearner'